### PR TITLE
First review of the Kanidm Administration book

### PR DESCRIPTION
**Pull request by cjschroder**
Corrected some typos, a few grammar corrections, trimmed the line lengths in
    the example server configuration file because side scrolling in the
    published version

Fixes #

- [ ] cargo fmt has been run
- [ ] cargo clippy has been run
- [ ] cargo test has been run and passes
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: kanidm_book/src/installing_the_server.md [hunk 1]
```diff
@@ -62,7 +64,7 @@ You'll need a volume where you can place configuration, certificates, and the da
     docker volume create kanidmd
 
 You should have a chain.pem and key.pem in your kanidmd volume. The reason for requiring
-TLS is explained in [why tls](./why_tls.md). In summary, TLS is our root of trust between the
+Transport Layer Security (TLS, which replaces the deprecated Secure Sockets Layer, SSL)) is explained in [why tls](./why_tls.md). In summary, TLS is our root of trust between the
```
Comment (yaleman) [inline kanidm_book/src/installing_the_server.md:67]: Duplicate closing bracket's been added here.

File: kanidm_book/src/intro.md [hunk 2]
```diff
@@ -17,11 +17,15 @@ The intent of the Kanidm project is to:
 
 Looking for the `rustdoc` documentation for the libraries themselves? [Click here!](./rustdoc/master/kanidm/)
 
+TODO This link does not work. Is this the correct URL? https://doc.rust-lang.org/rustdoc/index.html
```
Comment (yaleman) [inline kanidm_book/src/intro.md:20]: It should point to https://kanidm.com/documentation/

File: kanidm_book/src/server_configuration.md [hunk 3]
```diff
@@ -2,76 +2,97 @@
...
+    #   to match. Valid choices are:
     #   [zfs, other]
-    #   If you are unsure about this leave it as the default (other). After changing this
+    #   If you are unsure about this leave it as the default 
+    #   (other). After changing this
     #   value you must run a vacuum task.
     #   - zfs:
-    #     * sets sqlite pagesize to 64k. You must set recordsize=64k on the zfs filesystem.
+    #     * sets sqlite pagesize to 64k. You must set 
+    #   recordsize=64k on the zfs filesystem.
```
Comment (yaleman) [inline kanidm_book/src/server_configuration.md:28]: For this and similar points below - if we're going to line break something two nested indents deep, can we please indent them to align with its parent line? It's a little hard to follow otherwise.

File: kanidm_book/src/installing_the_server.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # Installing the Server
 
-> **NOTE** Our preferred deployment method is in containers, the documentation assumes you're running in docker. Kanidm will run in traditional compute, and server builds are available for multiple platforms or you can build the binaries yourself.
+> **NOTE** Our preferred deployment method is in containers, the documentation assumes you're running in docker. Kanidm will run in traditional compute, and server builds are available for multiple platforms, or you can build the binaries yourself.
+
+TODO why are containers preferred?
```
Comment (yaleman) [inline kanidm_book/src/installing_the_server.md:5]: @Firstyear could answer this one 😄

**Review by yaleman — CHANGES_REQUESTED** [review]
Thanks! A few comments added along the way.

File: kanidm_book/src/installing_the_server.md (hunk 4 above)
Comment (Firstyear) [reply kanidm_book/src/installing_the_server.md:5]: Because packaging the kanidm server for most distros is a shit show. 🙃 It's a lot easier if we say "hey use a container" then we at least have a smaller surface area for things to go wrong.

**Comment by Firstyear** [comment]
Resolved the comments as @cjschroder indicated she's not available for a bit.

**Review by Firstyear — APPROVED** [review]

**Comment by Firstyear** [comment]
@yaleman I'll leave you to approve and squash these :) 

I need to update contributors in a seperate PR too.

**Review by yaleman — APPROVED** [review]
